src/basics.md

```diff
@@ -1385,6 +1386,44 @@ fn main() {
 }
 ```
 
+[ex-examine-datetime]: #ex-examine-datetime
+<a name="ex-examine-datetime"></a>
+## Examine a DateTime struct via Datelike and Timelike traits
```

Comment:
I'd probably avoid using the tpe and trait names in the example title. How about something along the lines of "Examine the date and time"?

Reply:
Sounds good! I was trying to be explicit toward the issue, but you are right that including it makes it a little verbose/wordy. Thanks for the review